src/panel/NoResultMessage.jsx

```diff
@@ -0,0 +1,20 @@
+/* globals _ */
+import React from 'react';
+
+const NoResultMessage = () => {
+  return (
+    <>
+      <p className="u-center u-mb-xs u-text--smallTitle">
+        {_('Sorry, we could not find this place 🏝️', 'suggest')}
+      </p>
+      <p className="u-center u-text--subtitle u-mb-l">
```

Comment:
I think we don't need this bottom margin until we display something under the message. It looks a bit odd, I would remove the `u-mb-l` entirely.